uc-crux-llvm/src/UCCrux/LLVM/Overrides/Check.hs

```diff
@@ -168,23 +168,24 @@ createCheckOverride appCtx modCtx usedRef argFTys constraints cfg funcSym =
              llvmOverride_args = Crucible.cfgArgTypes cfg,
              llvmOverride_ret = Crucible.cfgReturnType cfg,
              llvmOverride_def =
-               \mvar bak args ->
```

Comment:
Note to reviewers that I didn't change the content of any of these large blocks, Github is just bad at displaying diffs that involve indenting blocks like this.